packages/wrangler/src/deployment-bundle/bundle.ts

```diff
@@ -305,6 +307,25 @@ export async function bundleWorker(
 
 		inject.push(checkedFetchFileToInject);
 	}
+
+	// When multiple workers are running we need some way to disambiguate logs between them. Inject a patched version of `globalThis.console` that prefixes logs with the worker name
+	if (getFlag("MULTIWORKER")) {
+		const patchedConsoleFileToInject = path.join(
+			tmpDir.path,
+			"patch-console.js"
+		);
+
+		if (!fs.existsSync(patchedConsoleFileToInject)) {
+			fs.writeFileSync(
+				patchedConsoleFileToInject,
+				fs.readFileSync(
+					path.resolve(getBasePath(), "templates/patch-console.js")
+				)
+			);
+		}
+
+		inject.push(patchedConsoleFileToInject);
+	}
```

Comment:
Can this be done inside middleware so that we don't have to hard-code the code path in the main bundle handler?

(sorry this wasn't in the first review)

Reply:
https://github.com/cloudflare/workers-sdk/pull/7251/commits/4f2b07044fd5a24d5e626a80c8f02e8387203898